docs/content/Tutorial:-A-First-Look-at-Druid.md

```diff
@@ -72,13 +78,13 @@ You should see a bunch of files:
 Running Example Scripts
 -----------------------
 
-Let's start doing stuff. You can start a Druid [Realtime](Realtime.html) node by issuing:
+Let's start doing stuff. You can start an example Druid [Realtime](Realtime.html) node by issuing:
 
 ```
 ./run_example_server.sh
 ```
 
-Select "wikipedia".
+Select "2" for the "wikipedia" example.
```

Comment:
That form still works :) actually, when specified as an arg to the script like that, it will look for examples/{name}/{name}_runtime.properties the same as it did before. if that file isn't found it falls back to prompting again.